tests/main.js

```diff
@@ -163,14 +163,12 @@ testBundlers('Throws on invalid package.json', [ESBUILD, ESBUILD_ZISI, DEFAULT],
   const invalidPackageJsonDir = `${fixtureDir}/invalid-package-json`
   const srcPackageJson = `${invalidPackageJsonDir}/package.json.txt`
   const distPackageJson = `${invalidPackageJsonDir}/package.json`
-  const expectedErrorRegex =
-    bundler === DEFAULT ? /invalid JSON/ : /package.json:1:1: error: Expected string but found "{"/
 
   await pRename(srcPackageJson, distPackageJson)
   try {
     await t.throwsAsync(
       zipNode(t, 'invalid-package-json', { opts: { jsBundler: bundler }, fixtureDir }),
-      expectedErrorRegex,
+      /(invalid JSON|package.json:1:1: error: Expected string but found "{")/,
```

Comment:
Should we add a test case for Prisma?

Reply:
Sorry, I should've added some context about this change. It's not related to Prisma per se. It's about the fact that we might get a slightly different error message for when a `package.json` is invalid, depending on whether or not we have any external modules configured. This is varies with who validates the file: "us" or esbuild.

I haven't changed the Prisma test case because I can't think of a good way of mocking the `postinstall` scenario.

Reply:
Sounds good :+1: